replications/internal/queue_management_test.go

```diff
@@ -506,3 +527,34 @@ func closeRq(rq *replicationQueue) {
 	close(rq.done)
 	rq.wg.Wait() // wait for run() function to return
 }
+
+func TestGetReplications(t *testing.T) {
+	t.Parallel()
+
+	path, qm := initQueueManager(t)
+	defer os.RemoveAll(path)
+
+	// Initialize 3 queues (2nd and 3rd share the same orgID and localBucket)
+	require.NoError(t, qm.InitializeQueue(id1, maxQueueSizeBytes, orgID1, localBucketID1))
+	require.DirExists(t, filepath.Join(path, id1.String()))
+
+	require.NoError(t, qm.InitializeQueue(id2, maxQueueSizeBytes, orgID2, localBucketID2))
+	require.DirExists(t, filepath.Join(path, id1.String()))
+
+	require.NoError(t, qm.InitializeQueue(id3, maxQueueSizeBytes, orgID2, localBucketID2))
+	require.DirExists(t, filepath.Join(path, id1.String()))
+
+	// Should return one matching replication queue (repl ID 1)
+	repls := qm.GetReplications(orgID1, localBucketID1)
+	require.Equal(t, 1, len(repls))
+	require.Contains(t, repls, id1)
```

Comment:
Should be able to use `require.ElementsMatch` here to compare the result slices contain the same elements without regard for sorting...will be more useful on lines 558 & 559